src/encoder.rs

```diff
@@ -526,6 +527,7 @@ pub struct FrameInvariants<T: Pixel> {
   /// how much future frames depend on the block (for example, via
   /// inter-prediction).
   pub block_importances: Box<[f32]>,
+  pub cpu_feature_level: CpuFeatureLevel,
```

Comment:
This doesn't look like it belongs in `FrameInvariants`

Reply:
I'm going to move it to EncoderConfig. It no longer makes sense to put it in Config since I'm using env vars to override the default.